config/coaddBase.py

```diff
@@ -0,0 +1,10 @@
+# Configs shared between makeWarp and assembleCoadd
```

Comment:
The settings in this config file assumes warps and coadds are being built for DRP reasons and not for AP template reasons; I suggest setting these in `DRP.yaml` instead of here (presumably in the `makeWarp` task).

Reply:
All configs in the files `coaddBase.py`, `makeWarp.py`, `compareWarpAssembleCoadd.py` and `assemblyCoadd.py` have been moved to config sections in the main `DRP.yaml` pipeline file. This requires some amount of config duplication, as several of these configs are shared between quanta. I ran before/after diffs on the config outputs to confirm that the configs are identical.